Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/RenderViewportWidget.cpp

```diff
@@ -222,15 +222,28 @@ namespace AtomToolsFramework
         switch (event->type()) 
         {
             case QEvent::Resize:
+            case QEvent::ShowToParent: //This event exists to capture the case where a resize event is missed "after" the underlying surface is modified by Qt (Seen during level load in Editor)
```

Comment:
Nitpick: This comment is really long (but is very useful!), could you please break it up only multiple lines?

Reply:
I missed this comment. I will update this as part of another PR.